.github/workflows/e2e.yml

```diff
@@ -76,6 +76,10 @@ jobs:
           make container-build BUILDER=docker IMG=${{ env.OPERATOR_IMAGE_NAME }}
           kind load docker-image ${{ env.OPERATOR_IMAGE_NAME }}
 
+      - name: Build and Install SonataFlow worfklow test images
+        run: |
+          make workflow_test_image_build-and-push
```

Comment:
This is unnecessary. The e2e `BeforeSuite` uses the operator to deploy three workflows to build images within the cluster. At the end of the build, the suite will fetch the images from the `SonataFlowBuild` `.status.image`. 

During the e2e, we replace the `e2e/testdata` YAML `.spec.podTemplate.container.image` attribute. So that when the operator deploys the `SonataFlow` custom resource the image is already there in the registry.

If you were to add a new e2e make sure to use the [kubectlPatchSonataFlowImageAndRollout](https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/main/test/e2e/helpers.go#L97) to replace with the right image tag:

```go
type E2EWorkflows struct {
	// CallBackPersistence is the prebuilt workflow to use on every use case that requires persistence
	CallBackPersistence E2EWorkflowEntry
	// CallBack is the prebuilt workflow to use on every use case that requires events or jobs services integration
	CallBack E2EWorkflowEntry
	// Greetings is the general usage image to use on any use case that requires a SonataFlow deployment
	Greetings E2EWorkflowEntry
}
```